### PR TITLE
[ADD] elico_crm_extra

### PR DESCRIPTION
**Pull request by noahzaozao**
@elicoidal @seb-elico This module is not installed in our ERP , move it to here.

### CODE REVIEW

File: elico_crm_extra/__init__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```
Comment (seb-elico) [inline elico_crm_extra/__init__.py]: Can you use short headers please?

File: elico_crm_extra/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```
Comment (seb-elico) [inline elico_crm_extra/__openerp__.py]: Can you use short headers please?

File: elico_crm_extra/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Elico CRM',
```
Comment (seb-elico) [inline elico_crm_extra/__openerp__.py]: Let's rename it to `CRM period filters` (and the module name: `crm_period_filters`)
@elicoidal do you confirm? Should we keep this module in the first place?

**Review by seb-elico — CHANGES_REQUESTED** [review]

**Comment by elicoidal** [comment]
Seems very specific. I would keep it private Elico @seb-elico @victormartinelicocorp @noahzaozao

**Comment by seb-elico** [comment]
@elicoidal it might be very specific but since we don't use it (not installed in our ERP), IMO we can leave it in our public addons to keep it somewhere

**Comment by noahzaozao** [comment]
@elicoidal @seb-elico I moved this PR to https://github.com/Elico-Corp/elico_odoo/pull/51

**Comment by seb-elico** [comment]
@noahzaozao based on my last comment, I'm reopening it and closing Elico-Corp/elico_odoo#51

**Comment by seb-elico** [comment]
LGTM FT

**Pull request merged**